tsdb/head.go

```diff
@@ -164,6 +164,10 @@ type HeadOptions struct {
 	// The default value is GOMAXPROCS.
 	// If it is set to a negative value or zero, the default value is used.
 	WALReplayConcurrency int
+
+	// Maximum number of CPUs that can be used for sorting postings lists.
+	// The default value is 0, meaning all available CPU cores will be used.
+	MemPostingsEnsureOrderConcurrency int
```

Comment:
Shouldn't be propagated to TSDB options? How can you set this when opening a TSDB?

Reply:
You are right, I will add this soon.